docs/docs/how-plugins-work.md

```diff
@@ -3,77 +3,3 @@ title: How Plugins Work
 ---
 
 You may be looking to build a plugin that doesn't exist yet, or you may just be curious to know more about the anatomy of a Gatsby plugin. We'll review:
```

Comment:
Is this page still planned on being used or is it being kept for the redirect purposes?

Reply:
I think it can be deleted!